systemd/datadog_checks/systemd/data/conf.yaml.example

```diff
@@ -0,0 +1,10 @@
+init_config:
+
+instances:
```

Comment:
Since we're also grabbing a bunch of data about systemd itself, in addition to units, the config should be a bit different I think. We only wanna grab all data once. So it should be a single instance that has a list in it that contains all the units:

```yaml
instance:
  units:
  - unit_name: networking
    unit_id: networking.service
  - unit_name: cron
    unit_id: cron.service
  - unit_name: ssh
    unit_id: ssh.service
```

Reply:
I'm having an issue with that config if I do `units = instance.get('units', [])`:

```
cannot unmarshal !!map into []integration.RawMap
```

Which I believe refers to: https://github.com/DataDog/datadog-agent/blob/6.8.x/pkg/autodiscovery/integration/config.go#L24 ?

Reply:
Updated from dict to array for `instances`